LayoutTests/http/wpt/mediarecorder/MediaRecorder-AV-audio-video-dataavailable.html

```diff
@@ -115,9 +115,20 @@
         await waitForEvent(player, 'seeked');
         resContext.drawImage(player, 0, 0);
         _assertPixelApprox(resFrame, 25, 25, 255, 0, 0, 255, "25, 25", "255, 0, 0, 255", 50);
-        _assertPixelApprox(resFrame, 50, 50, 255, 0, 0, 255, "50, 50", "255, 0, 0, 255", 50);
-    }, 'MediaRecorder can successfully record the video for a audio-video stream');
+        _assertPixelApprox(resFrame, 50, 50, 255, 0, 0, 255, "50, 50", "255, 0, 0, 255", 50);        
+    }
+    
+    promise_test(async (test) => {
+        return runTest('video/mp4');
+    }, 'MediaRecorder can successfully record the video for a audio-video (aac) stream');
 
+    promise_test(async (test) => {
+        return runTest('video/mp4; codecs=avc1.42000a,pcm');
+    }, 'MediaRecorder can successfully record the video for a audio-video (pcm) stream');
+
+    promise_test(async (test) => {
+        return runTest('video/mp4; codecs=avc1.42000a,alac');
```

Comment:
Should we add a feature flag?

Reply:
I don't see the need here, it's a well defined path using officially supported encoder/decoder already